helm/openig/templates/tls.yaml

```diff
@@ -1,20 +1,22 @@
-{{- if .Values.useTLS }}
+# CertManager certificate object created if 'tls.strategy: CertManager'. 
```

Comment:
See my suggestion in the README for replacing this comment.

Reply:
done